rabbitmq-http-connector/main.go

```diff
@@ -44,7 +44,11 @@ func (conn rabbitMQConnector) consumeMessage() {
 	}
 
 	forever := make(chan bool)
-	sem := make(chan int, 10) // Process maximum 10 messages concurrently
+	concurrent := os.Getenv("CONCURRENT")
```

Comment:
```suggestion
        concurrent := 0
 	 if os.Getenv("CONCURRENT") != ""{
	 concurrent = os.Getenv("CONCURRENT")
	 }
```
You should probably do something like that. Thanks for contributing but can you please test out the PR once before pushing further changes?

Reply:
Sure, will do it at my local. And sorry to send a PR without testing. 🙇‍♂️